components/Login.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { useRouter } from 'next/router';
+import { useForm } from 'react-hook-form';
+import useHydration from '../hooks/useHydration';
+import styled from 'styled-components';
+import NavWrapper from './NavWrapper';
+import { LoginButton } from './Buttons';
+
+import Toast from './Toast';
+
+export default function Login() {
+  const hydrated = useHydration();
+  const [open, setOpen] = useState(false);
+  const router = useRouter();
+
+  const { register, handleSubmit, getValues } = useForm();
+
+  // START ---------------------  This is a temporary login solution //
+
+  const onSubmit = () => {
+    const user = getValues('user');
+    const password = getValues('password');
+    if (password === '1234' && user === 'Tom') {
+      router.push('/shopping');
+    } else {
+      setOpen(true);
+      setTimeout(() => {
+        setOpen(false);
+      }, 2500);
+    }
+  };
+
+  // END ---------------------  This is a temporary login solution //
```

Comment:
I think implementing a temporary solution to showcase a login feature like this is a clever move!